components/browser/toolbar/README.md

```diff
@@ -34,9 +34,15 @@ This component emits the following [Facts](../../support/base/README.md#Facts):
 
 | Action | Item    | Extras         | Description                        |
 |--------|---------|----------------|------------------------------------|
-| CLICK  | menu    |                | The user opened the overflow menu. |
+| CLICK  | menu    | `menuExtras`   | The user opened the overflow menu. |
 | COMMIT | toolbar | `commitExtras` | The user has edited the URL.       |
 
+#### `menuExtras`
+
+| Key          | Type    | Value                                 |
+|--------------|---------|---------------------------------------|
+| custom_tab   | Boolean | Whether the menu is from a custom tab |
+
```

Comment:
I think we can remove this here now since that is not really added by the `browser-toolbar` component anymore.